modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -3916,6 +3984,7 @@ def _compute_new_widths():
                 new_index = self.index.append([other.index for other in others])
             new_columns = joined_index
             frames = [self] + others
+            # TODO: should we wrap all `concat` call into "try except" block?
```

Comment:
Why?

Reply:
This function can throw exception in case of duplicate values

Reply:
Let's expand this comment on this matter.